okhttp-logging-interceptor/README.md

```diff
@@ -0,0 +1,44 @@
+Logging Interceptor
+===================
+
+An [OkHttp interceptor][1] which logs HTTP request and response data.
```

Comment:
Add a note that `HEADERS` and `BODY` levels shouldn't be used in production? Otherwise secrets are leaked. (`Authorization`, `Cookie`, etc.)